packages/component/src/ActivityFeedback/providers/useFeedbackText.ts

```diff
@@ -0,0 +1,6 @@
+import { type Dispatch, type SetStateAction } from 'react';
+import useActivityFeedbackContext from './private/useActivityFeedbackContext';
+
+export default function useFeedbackText(): readonly [string, Dispatch<SetStateAction<string>>] {
+  return useActivityFeedbackContext().feedbackTextState;
```

Comment:
Again, context trapping. We need to talk more about this.

Reply:
Let's experiment something new in this PR.